src/CoreArbiterServer.cc

```diff
@@ -1579,6 +1590,11 @@ CoreArbiterServer::installSignalHandler() {
         LOG(ERROR, "Couldn't set signal handler for SIGSEGV");
     if (sigaction(SIGABRT, &signalAction, NULL) != 0)
         LOG(ERROR, "Couldn't set signal handler for SIGABRT");
+
+    // Set sa_flags to 0 to ensure the function fail
```

Comment:
Why do want to "ensure the function fail?" Also, which function are we trying to ensure fails?